cluster/gce/config-test.sh

```diff
@@ -72,7 +72,7 @@ fi
 # Also please update corresponding image for node e2e at:
 # https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/image-config.yaml
 CVM_VERSION=${CVM_VERSION:-container-vm-v20170627}
-GCI_VERSION=${KUBE_GCI_VERSION:-cos-stable-63-10032-71-0}
+GCI_VERSION=${KUBE_GCI_VERSION:-cos-dev-66-10452-8-0}
```

Comment:
Why do we use m66 here? IIRC, this is used for gce cluster e2e tests where users want to use a default image.

Reply:
I was talking to Adam about this... Considering we want to launch GKE 1.10 using COS m66, we thought it would be good to get this in the open-source release so it starts getting some testing already... What do you think? We can talk about this in person if you'd like.

Reply:
Just discussed this with @yguo0905 and @yujuhong and it seems we're fine with pushing m66 here...